things/service.go

```diff
@@ -247,13 +242,42 @@ func (ts *thingsService) UpdateThing(ctx context.Context, token string, thing Th
 	return ts.things.Update(ctx, thing)
 }
 
+func (ts *thingsService) ShareThing(ctx context.Context, token, thingID string, policies, userIDs []string) error {
```

Comment:
Is `policies` parameter named ambiguously? It represents the action, not a list of `policies` per se, it's not "policy ids" or "policy names"?

Reply:
Yes, it represents actions related to Things policies: `access`, `write` and `delete`. However, in the essence, they are `relations`, not policies. I'll update the naming.